app/controllers/v3/service_instances_controller.rb

```diff
@@ -217,6 +217,16 @@ def parameters
     end
   end
 
+  def show_permissions
+    service_instance = ServiceInstance.first(guid: hashed_params[:guid])
+    service_instance_not_found! unless service_instance
+
+    render status: :ok, json: {
+      manage: is_space_active?(service_instance.space) ? can_write_to_active_space?(service_instance.space) : permission_queryer.can_write_globally?,
```

Comment:
There is a helper method `admin?` defined below that can be used instead of `permission_queryer.can_write_globally?`.

Reply:
Yeah fair enough